### ISSUE TITLE
Can't reproduce the reported results

### ISSUE THREAD

**Issue by bt-nghia** [Xiaohao-Liu/BundleGT#1]
Dear authors, i had follow your instructions in paper and reproduce with your provided code but it didn't give me the same result as reported in the paper:

Youshu is run on GPU T4, NetEase and iFashion are run on P100
With Youshu i run the script you provide and had, TOP 20: REC_T=0.27914, NDCG_T=0.16419.
With NetEase i set the HGT layer to 5 and \alpha_1 = \alpha_2 = 0.5. Recall@20=0.07046, NDCG@20=0.03666.
With iFashion i set the HGT layer to 1 and \alpha_1 = \alpha_2 = 0.75. TOP 20: REC_T@20=0.04623, NDCG_T@20=0.03220.

The notebook available at 
[iFashion_reproduceBundleGT](https://www.kaggle.com/code/namnh2711/reproducebundlegt/notebook)

Thank for support!

**Comment by Xiaohao-Liu** [comment]
Thanks for your interest in BundleGT! We were using V100 and A5000 to conduct the experiments. For ease of reproducibility, we provide the detailed hyperparameter settings for each dataset as follows:

> Youshu:
> {'data_path': './datasets', 'batch_size_train': 2048, 'batch_size_test': 2048, 'topk': [10, 20, 40, 80], 'neg_num': 1, 'test_interval': 10, 'dataset': 'Youshu', 'model': 'BundleGT', 'embedding_size': 64, 'info': '', 'folder': '', 'lr': 0.001, 'l2_reg': 1e-05, 'early_stopping': 40, 'num_ui_layers': 4, 'num_trans_layers': 3, 'gcn_norm': False, 'num_trim': 70, 'ub_alpha': 0.5, 'bi_alpha': 0.5}

> NetEase:
> {'batch_size_train': 2048, 'batch_size_test': 2048, 'topk': [10, 20, 40, 80], 'neg_num': 1, 'test_interval': 5, 'dataset': 'NetEase', 'model': 'BundleGT', 'embedding_size': 64, 'info': '', 'folder': '', 'lr': 0.001, 'l2_reg': 1e-06, 'early_stopping': 40, 'num_ui_layers': 4, 'num_trans_layers': 5, 'gcn_norm': False, 'num_trim': 70, 'ub_alpha': 0.5, 'bi_alpha': 0.5}

> iFashion:
> {'batch_size_train': 2048, 'batch_size_test': 2048, 'topk': [10, 20, 40, 80], 'neg_num': 1, 'test_interval': 2, 'dataset': 'iFashion', 'model': 'BundleGT', 'embedding_size': 64, 'info': '', 'folder': '', 'lr': 0.001, 'l2_reg': 2e-06, 'early_stopping': 20, 'num_ui_layers': 4, 'num_trans_layers': 1, 'gcn_norm': True, 'num_trim': 5, 'transformer': '', 'ub_alpha': 0.75, 'bi_alpha': 0.75}

From your notebook, it seems like you did not consider the impact of learning rate, L2 and etc. As for Youshu, some teams also sent me emails asking about the hyperparameter settings on NetEase and iFashion, and mentioned that corresponding results can be successfully obtained on Youshu. So it may be the issue of machines in your case.

**Comment by bt-nghia** [comment]
Thank for your support!
